ArduCopter/mode_althold.cpp

```diff
@@ -32,11 +32,11 @@ void ModeAltHold::run()
     update_simple_mode();
 
     // get pilot desired lean angles
-    float target_roll, target_pitch;
-    get_pilot_desired_lean_angles(target_roll, target_pitch, copter.aparm.angle_max, attitude_control->get_althold_lean_angle_max_cd());
+    float target_roll_cd, target_pitch_cd;
+    get_pilot_desired_lean_angles_cd(target_roll_cd, target_pitch_cd, copter.aparm.angle_max, attitude_control->get_althold_lean_angle_max_cd());
 
     // get pilot's desired yaw rate
-    float target_yaw_rate = get_pilot_desired_yaw_rate();
+    float target_yaw_rate_cds = get_pilot_desired_yaw_rate_cds();
 
     // get pilot desired climb rate
     float target_climb_rate = get_pilot_desired_climb_rate();
```

Comment:
not a blocker but in a future PR we could rename the variable to "target_climb_rate_cms" both here in althold and also in a few other modes (auto, autotune, etc)